ui/app/components/currency-display/currency-display.container.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+import CurrencyDisplay from './currency-display.component'
+import { getValueFromWeiHex, formatCurrency } from '../../helpers/confirm-transaction/util'
+import { ETH } from '../../constants/common'
+
+const mapStateToProps = (state, ownProps) => {
+  const { value, numberOfDecimals = 2, currency } = ownProps
+  const { metamask: { currentCurrency, conversionRate } } = state
+
+  const toCurrency = currency === ETH ? ETH : currentCurrency
```

Comment:
Based on the use cases of this component added in this PR, this line could be simplified to: `const toCurrency = currency || currentCurrency`

Reply:
Yeah my intention for this was so if anything but `'ETH'` got passed into `currency` that the component would use `currentCurrency` instead of what got passed in. I believe I can just use `propTypes` for that though, so I'll change this.